tests/lambda_endpoints/conftest.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import boto3
+import pytest
+from moto import mock_s3
+
+
+@pytest.fixture()
+def _aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+    os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
+
+
+@pytest.fixture()
+def s3_client(_aws_credentials):
+    with mock_s3():
```

Comment:
Yes, any tests that have `def test_name(s3_client):` in them would use this fixture and therefore get the mock. I also added another "autouse" fixture that relies on this one so that all of the lambda_endpoints get this during the setup step without needing to think about it. It is perhaps hiding a bit too much, but with the number of tests we had defined it seemed nicer to do this and make sure we were getting mocks all the time rather than a potential mix in the future.